jfr-streaming/src/main/java/io/opentelemetry/contrib/jfr/metrics/internal/cpu/OverallCPULoadHandler.java

```diff
@@ -30,36 +30,37 @@
   private static final String METRIC_NAME = "runtime.jvm.cpu.utilization";
   private static final String DESCRIPTION = "CPU Utilization";
 
-  private final Meter otelMeter;
   private BoundDoubleHistogram userHistogram;
   private BoundDoubleHistogram systemHistogram;
   private BoundDoubleHistogram machineHistogram;
 
-  public OverallCPULoadHandler(Meter otelMeter) {
-    this.otelMeter = otelMeter;
+  public OverallCPULoadHandler() {
+    initializeMeter(NoopMeter.getInstance());
   }
 
   @Override
-  public OverallCPULoadHandler init() {
-    var attr = Attributes.of(ATTR_CPU_USAGE, USER);
-    var builder = otelMeter.histogramBuilder(METRIC_NAME);
-    builder.setDescription(DESCRIPTION);
-    builder.setUnit(PERCENTAGE);
-    userHistogram = builder.build().bind(attr);
-
-    attr = Attributes.of(ATTR_CPU_USAGE, SYSTEM);
-    builder = otelMeter.histogramBuilder(METRIC_NAME);
-    builder.setDescription(DESCRIPTION);
-    builder.setUnit(ONE);
-    systemHistogram = builder.build().bind(attr);
-
-    attr = Attributes.of(ATTR_CPU_USAGE, MACHINE);
-    builder = otelMeter.histogramBuilder(METRIC_NAME);
-    builder.setDescription(DESCRIPTION);
-    builder.setUnit(ONE);
-    machineHistogram = builder.build().bind(attr);
-
-    return this;
+  public void initializeMeter(Meter meter) {
+    userHistogram =
+        meter
+            .histogramBuilder(METRIC_NAME)
+            .setDescription(DESCRIPTION)
+            .setUnit(PERCENTAGE)
+            .build()
+            .bind(Attributes.of(ATTR_CPU_USAGE, USER));
+    systemHistogram =
+        meter
+            .histogramBuilder(METRIC_NAME)
+            .setDescription(DESCRIPTION)
+            .setUnit(ONE)
```

Comment:
Noticed that user CPU usage unit is percentage while system and machine CPU usage unit is "1". Seems off.

Reply:
the unit for user CPU usage was just updated in #140, probably just missed it for system and machine